docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+
+The longhand syntax accepts an array of objects, each object containing the `id` of the entry to be connected and an optional `position` object to define where to add the relation.
+
+::: note Different syntaxes for different relations
+The syntaxes described in this documentation are useful for one-to-many, many-to-many and many-ways relations.<br />For one-to-one, many-to-one and one-way relations, the syntaxes are also supported but only the last relation will be used, so it's preferable to use a shorter format (e.g.: `{ data: { category: 2 } }`, see [REST API documentation](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md#requests)).
+:::
+
+To define the `position` for a relation, pass one of the following 4 different positional attributes:
+
+| Parameter name and syntax | Description                                                            | Type       |
+| ------------------------- | ---------------------------------------------------------------------- | ---------- |
+| `before: id`              | Positions the relation before the given `id`.                          | Entry `id` |
+| `after: id`               | Positions the relation after the given `id`.                           | Entry `id` |
+| `start: true`             | Positions the relation at the start of the existing list of relations. | Boolean    |
+| `end: true`               | Positions the relation at the end of the existing list of relations.   | Boolean    |
+
+The `position` argument is optional and defaults to `position: { end: true }`.
+
+:::note Sequential order
+Since `connect` is an array, the order of operations is important as they will be treated sequentially (see combined example below).
+:::
+
+::::: tabs card
+
+:::: tab Basic example
+
+Consider the following record in the database:
+
+```json
+categories: [
+  { id: 1, order: 1 }
+  { id: 2, order: 2 }
+]
+```
+
+Sending the following request updates a `restaurant` entry and creates a third relation of `id` `3` for the `categories` attribute and positions it before the relation with `id` `2`:
+
+::: request Example request to update the position of one relation
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: {
+      connect: [
+        { id: 3, position: { before: 2 } },
+      ]
+    }
+  }
+}
+```
+
+:::
+::::
+
+:::: tab Combined example
+
+Consider the following record in the database:
+
+```json
+categories: [
+  { id: 1, order: 1 }
+  { id: 2, order: 2 }
+]
+```
+
+Sending the following example in the request body of a PUT request updates multiple relations:
+
+::: request Example request to reorder several relations
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: {
+      connect: [
+        { id: 6, position: { after: 1} },
+        { id: 7, position: { before: 2 } },
+        { id: 8, position: { end: true } },
+        { id: 9 },
+        { id: 10, position: { start: true } },
```

Comment:
I don't know if it's worth mentioning, but the same relation should not be connected more than once.
That would return a Validation error by the API.